qiita_pet/support_files/doc/source/europeanbioinformaticsinstitute.rst

```diff
@@ -31,41 +33,63 @@ from private to public, so consider this when submitting data and your manuscrip
 
 .. note::
    For convenience Qiita allows you to upload a QIIME mapping file to process your data. However,
-   the QIIME mapping file, in general, doesn't have all the EBI/ENA fields. Thus, you will need to
+   the QIIME mapping file, in general, does not have all the EBI/ENA fields. Thus, you will need to
    update your information files (sample or preparation) via the update option. To simplify this process,
    you can download the system generated files and add/modify these fields for each file.
 
 
 EBI-ENA NULL values vocabulary
 ------------------------------
 
-We support the following values: *not applicable*, *not collected*, *not provided*, *restricted access*.
+We support the only following values: *not applicable*, *not collected*, *not provided*, *restricted access*.
 
 For the latest definitions and explanation visit the `EBI/ENA Missing value reporting <http://www.ebi.ac.uk/ena/about/missing-values-reporting>`__.
 
 .. warning::
    Column names in your information files cannot be named as a Postgres reserved word. For example, a column cannot be named `CONDITION`, but could instead be named `DISEASE_CONDITION`. For a full list of these reserved words, see this `link <https://www.postgresql.org/docs/9.3/static/sql-keywords-appendix.html>`__.
 
-
 Required Sample Information Fields for EBI submission
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+These are the columns required for successfully submit any data to EBI via Qiita:
```

Comment:
These blocks - any field listing might be better as a table ... thoughts?

Reply:
Yes, I agree and this may be doable for the following week since we're pushing the deadline. It was a huge PITA last night to try to format the table with the explanations, though I can also see the advantage to just having it as a block so they are encouraged to use either Qiimp or Gail's template.